blackbox/docs/admin/system-information.rst

```diff
@@ -602,6 +602,32 @@ in subsequent versions. All ``LONG`` columns always return ``0``.
 |                                                        | See `tools.ietf.org <https://tools.ietf.org/html/rfc2525#page-50>`_.                       |             |
 +--------------------------------------------------------+--------------------------------------------------------------------------------------------+-------------+
 
+``connections``
+---------------
+
++---------------------------------------+-------------------+-------------------+
+| Column Name                           | Description       | Return Type       |
++=======================================+===================+===================+
+| ``http``                              | Number of         | ``OBJECT``        |
+|                                       | connections       |                   |
+|                                       | established via   |                   |
+|                                       | HTTP              |                   |
++---------------------------------------+-------------------+-------------------+
+| ``http[currently_open]``              | The currently     | ``LONG``          |
```

Comment:
I'd prefer `active` or wouldn't that be accurate?

Reply:
is an open connection always active?

Reply:
I think `open connections` is a more popular terminology.